mantidproject/mantid
# Convert Group 9 and 11 from Python 2 to Python 3

This PR deals with the Python2 to Python3 conversion of things in `scripts/Interface/` towards #18550.

**To test:**

1. Code review
1. Go to Interfaces -> Diffraction -> Powder Diffraction GUI
1. Confirm that there are no import or general errors when opening the interfaces
1. Play around with the interfaces (no data needed)
1. Confirm that there are no Python related errors

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
